TestsCalculator/SettingsPageTests.cs

```diff
@@ -0,0 +1,199 @@
+            Assert.IsTrue(settingsPage.DateFormatText.Displayed);
+            Assert.IsTrue(settingsPage.NumberFormatText.Displayed);
+            Assert.IsTrue(settingsPage.CurrencyFormatText.Displayed);
+        }
+
+        [Test]
+        public void CheckDateFormatOptions()
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+            List<string> expectedDateFormat = new List<string> { "dd/MM/yyyy", "dd-MM-yyyy", "MM/dd/yyyy", "MM dd yyyy" };
+
+            //Act
+            SelectElement s = settingsPage.DateFormatOptions;
+
+            IList<string> actualDateFormat = new List<string>();
+            for (int j = 0; j < s.Options.Count; j++)
+            {
+                actualDateFormat.Add(s.Options.ElementAt(j).Text);
+            }
+            //Assert
+            Assert.AreEqual(expectedDateFormat, actualDateFormat);
+        }
+
+        [Test]
+        public void CheckNumberFormatOptions()
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+            List<string> expectedNumberFormat = new List<string> { "123,456,789.00", "123.456.789,00", "123 456 789.00", "123 456 789,00" };
+
+            //Act
+            SelectElement s = settingsPage.NumberFormatOptions;
+            IList<string> actualNumberFormat = new List<string>();
+            for (int j = 0; j < s.Options.Count; j++)
+            {
+                actualNumberFormat.Add(s.Options.ElementAt(j).Text);
+            }
+
+            //Assert
+            Assert.AreEqual(expectedNumberFormat, actualNumberFormat);
+        }
+
+        [Test]
+        public void CheckCurrencyFormatOptions()
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+            List<string> expectedCurrencyFormat = new List<string> { "$ - US dollar", "€ - euro", "£ - Great Britain Pound" };
+
+            //Act
+            SelectElement s = settingsPage.CurrencyFormatOptions;
+            IList<string> actualCurrencyFormat = new List<string>();
+            for (int j = 0; j < s.Options.Count; j++)
+            {
+                actualCurrencyFormat.Add(s.Options.ElementAt(j).Text);
+            }
+
+            //Assert
+            Assert.AreEqual(expectedCurrencyFormat, actualCurrencyFormat);
+        }
+
+        [Test]
+        public void CancelBtnOpensDepositPage()
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+
+            //Act
+            settingsPage.CancelBtn.Click();
+
+            //Assert
+            string actualUrl = driver.Url;
+            string expectedUrl = "http://127.0.0.1:8080/Deposit";
+            Assert.AreEqual(expectedUrl, actualUrl);
+        }
+
+        [TestCase("MM dd yyyy", "M dd yyyy")]
+        [TestCase("MM/dd/yyyy", "M/dd/yyyy")]
+        [TestCase("dd-MM-yyyy", "dd-M-yyyy")]
+        [TestCase("dd/MM/yyyy", "dd/M/yyyy")]
```

Comment:
You may use the format twice. To select on Settings page and to convert date to string(format).
```suggestion
        [TestCase("MM dd yyyy")]
        [TestCase("MM/dd/yyyy")]
        [TestCase("dd-MM-yyyy")]
        [TestCase("dd/MM/yyyy")]
```

Reply:
corrected